upf/pfcp.c

```diff
@@ -1449,6 +1450,13 @@ encode_forwarding_policy (void *p, u8 ** vec)
   return 0;
 }
 
+void
+free_forwarding_policy (void *p)
```

Comment:
the type of the IE must also be changed to `SIMPLE_IE_FREE` for this to take effect.

Reply:
Fixed